### ISSUE TITLE
Runtime error with NNDC data

### ISSUE THREAD

**Issue by wbinventor** [ndpp/ndpp#11]
I tried running the ``ndpp/example/ndpp.xml`` example file with the NNDC data downloaded by OpenMC, and observed the following runtime error:

```
> /ndpp/examples$ ndpp

                 888b    888 8888888b.  8888888b.  8888888b.  
                 8888b   888 888  "Y88b 888   Y88b 888   Y88b 
                 88888b  888 888    888 888    888 888    888 
                 888Y88b 888 888    888 888   d88P 888   d88P 
                 888 Y88b888 888    888 8888888P"  8888888P"  
                 888  Y88888 888    888 888        888        
                 888   Y8888 888  .d88P 888        888        
                 888    Y888 8888888P"  888        888        
____________________________________________________________________________

      Developed At:  University of Michigan and
                     Massachusetts Institute of Technology
      Version:       1.0.0
      Git SHA1:      e26583d0471febd9333d5dbe1625836809af6032
      Date/Time:     2015-02-13 15:13:06

 ===========================================================================
 ========================>     INITIALIZATION     <=========================
 ===========================================================================

 Reading Nuclear Data Pre-Processor XML file...
 Reading cross sections XML file...
 Beginning Pre-Processing...
  Performing Pre-Processing For 1001.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 1002.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 1003.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 2003.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 2004.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 3006.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
     Inelastic Evaluation 0% Complete
     Inelastic Evaluation 5% Complete
     ...
     Inelastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 3007.71c
 ....Performing Scattering Integration
     Elastic Evaluation 0% Complete
     Elastic Evaluation 5% Complete
     ...
     Elastic Evaluation 100% Complete
     Inelastic Evaluation 0% Complete
     Inelastic Evaluation 5% Complete
     ...
     Inelastic Evaluation 100% Complete
 ....Performing Fission Neutron Energy Integration
  Performing Pre-Processing For 4007.71c
 ....Performing Scattering Integration
 ERROR: Value outside of array during binary search
```

I'm not sure if this is a known issue or bug with NDPP (or the NNDC data?), but given my lack of familiarity with NDPP, I thought I would post it here in the process of searching for a solution.

**Comment by nelsonag** [comment]
Interesting.  Be-7's data only goes up to 8.1MeV and this causes an out of bounds when searching the grid for 20 MeV.  Simple fix, though I thought I had a check in for just this case.  I'll have something shortly.

**Comment by nelsonag** [comment]
Resolved via PR #12
